content/documentation/pages/4-batch-developer-guides/3-troubleshooting/2-debugging-scdf-tasks.md

```diff
@@ -4,5 +4,146 @@ title: 'Debugging Batch applications deployed by Data Flow'
 description: 'Debugging Batch applications deployed by Data Flow'
 ---
 
-A Task is launched and the task fails. Now what?
-How to debug when using SCDF.
+# Debugging Tasks
```

Comment:
Let's also change this to "Debugging Tasks in SCDF" for consistency. We did it in Streams already.